components/popover/index.js

```diff
@@ -100,27 +96,22 @@ class Popover extends Component {
 			return;
 		}
 
-		const { content } = this.nodes;
-		if ( ! content ) {
+		if ( ! this.contentNode.current ) {
 			return;
 		}
 
+		// Without the setTimeout, the dom node is not being focused
```

Comment:
This is treating the symptom, not the cause, and the comment reads as such.

The real cause is a combination of:

- When initially mounted, the popover doesn't have a Slot context in which to render. When this becomes available, it re-renders the content into the slot (an unmount and remount operation).
- When initially mounted, the container has a style of `visibility: hidden`, so focusing would not select the container.

Reply:
See also: #8218